examples/docker/taxi-on-omnisci/build-docker-image.sh

```diff
@@ -20,5 +20,9 @@ It Can be generated by following the instructions on the link:
 
 cd "`dirname \"$0\"`"
 
+printf "Getting the small NYC Taxi dataset..."
```

Comment:
Is there a reason 
```suggestion
echo "Note: a user is responsible for preparing the dataset.
The dataset must be named as 'trips_xaa.csv' and be in the folder with 'nyc-taxi-omnisci.dockerfile'.
It Can be generated by following the instructions on the link:
'https://github.com/toddwschneider/nyc-taxi-data#instructions'"
```